src/i18n/translations/en/translation.json

```diff
@@ -123,6 +123,9 @@
   },
   "profile_page": {
     "mobileTitle": "Only available on Desktop",
-    "mobileDescription": "The Jumper Profile page is not available on small screens yet. We are working on it."
+    "mobileDescription": "The Jumper Profile page is not available on small screens yet. We are working on it.",
+    "rank": "Leaderboard is updated on a daily basis",
+    "pointsInfo": "Points is your score for interacting with Jumper. As you gain more points, your level goes up.",
+    "levelInfo": "A higher level increases your odds to win rewards from raffles, perks, partners, rewards and more."
```

Comment:
Personally, I think the writing can be improved to sound more professional:
ChatGPT threw me this ->

```
rank: 'The leaderboard is updated daily.',
pointsInfo: 'Points represent your score for engaging with Jumper. As you accumulate more points, your level increases.',
levelInfo: 'Reaching higher levels enhances your chances of winning rewards from raffles, perks, partners, and more.'
```